File: backend/InventoryService/app/domain/entities/product.py
class Product:
    def __init__(self, product_id, name, price, stock):
        self._product_id = product_id
        self._name       = name
        self._price      = price
        self._stock      = stock
        
    def getId(self):
        return self._product_id
    
    def getName(self):
        return self._name
    
    def getPrice(self):
        return self._price
    
    def getStock(self):
        return self._stock
    
    def __str__(self) -> str:
        return str({
            'product_id': self.getId(),
            'name': self.getName(),
            'price': self.getPrice(),
            'stock': self.getStock()
        })

File: backend/PaymentService/app/domain/entities/product.js
export class Product {
	#product_id
	#name
	#price
	#stock

	constructor(product_id, name, price, stock) {
		this.#product_id = product_id;
		this.#price = price;
		this.#name = name;
		this.#stock = stock;
	}

	getId() {
		return this.#product_id;
	}

	getName() {
		return this.#name;
	}

	getPrice() {
		return this.#price;
	}

	getStock() {
		return this.#stock;
	}
}